packages/core/src/utils.ts

```diff
@@ -21,15 +22,16 @@ export function isPolyMessage (message: string): boolean {
   return isPolyMessage;
 }
 
-export function observeAccounts (cb: (accounts: KeyringAccountData[]) => void) {
+export function observeAccounts (cb: (accounts: KeyringAccountData[]) => void): Subscription {
```

Comment:
this and the first `import` statement are the only real change to this file, the rest being linter stuff